packages/neuron-ui/src/components/AmendSUDTSend/index.tsx

```diff
@@ -7,7 +7,7 @@ import PageContainer from 'components/PageContainer'
 import Button from 'widgets/Button'
 import Spinner from 'widgets/Spinner'
 import { GoBack } from 'widgets/Icons/icon'
-import { scriptToAddress } from '@nervosnetwork/ckb-sdk-utils'
+import { scriptToAddress } from 'utils/scriptAndAddress'
```

Comment:
`addressToScript` and `scriptToAddress` is introduced from `utils` directly at https://github.com/nervosnetwork/neuron/pull/3149/files#diff-6039bcc6e52abc0b1130aa6f566193f6510eeca4037ea53599748f5d7e6b25d9R3, import statements could be unified.